charts/ingress-nginx/values.yaml

```diff
@@ -546,7 +546,11 @@ controller:
 
   opentelemetry:
     enabled: false
-    image: registry.k8s.io/ingress-nginx/opentelemetry:v20230312-helm-chart-4.5.2-28-g66a760794@sha256:40f766ac4a9832f36f217bb0e98d44c8d38faeccbfe861fbc1a76af7e9ab257f
+    image:
+      registry: registry.k8s.io
+      image: ingress-nginx/opentelemetry
+      tag: "v20230312-helm-chart-4.5.2-28-g66a760794"
+      digest: sha256:40f766ac4a9832f36f217bb0e98d44c8d38faeccbfe861fbc1a76af7e9ab257f
```

Comment:
Basically this is a breaking change, but it makes sense. Could you please also apply this to the `extraModules` array and do the whole concatenation in the `extraModules` template in the `_helpers.tpl`?

Reply:
thanks for the feedback!

I moved the concatenation to the `extraModules` template. I reused the existing template function for the digest.

I adapted the `extraModules` example in the values yaml to match the new pattern